todo-MVC-Kuzzle-VueJS/src/components/ManageList.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div id="ManageList">
+    <v-layout wrap align-center row>
+      <CreateNewList
+        v-if="modal"
+        :modal="modal"
+        @cancelClicked="onCancel"
+        @createClicked="onCreate"
+      />
+      <v-flex xs12 sm12 md6 class="text-xs-center">
+        <v-select
+          outline
+          label="Please choose a list"
+          item-text="text"
+          item-value="value"
+          :items="lists"
+          v-model="currentListCopy"
+          @input="currentListChanged"
+        />
+      </v-flex>
+      <v-flex xs12 sm12 md6 class="text-xs-center">
+        <v-btn color="blue" class="white--text" @click="newListClicked">
+          New List
+        </v-btn>
+      </v-flex>
+    </v-layout>
+  </div>
+</template>
+
+<script>
+import CreateNewList from './CreateNewList';
+
+export default {
+  name: 'ManageList',
+  components: {
+    CreateNewList
+  },
+  props: ['lists', 'currentList'],
+  watch: {
+    currentList: function handler(newVal, oldVal) {
```

Comment:
Why don't you just use an arrow function?
```suggestion
    currentList: (newVal, oldVal) => {
```